bot/constants.py

```diff
@@ -131,7 +131,12 @@ class Emojis:
     cross_mark = "\u274C"
     check = "\u2611"
     envelope = "\U0001F4E8"
-    trashcan = environ.get("TRASHCAN_EMOJI", "<:trashcan:637136429717389331>")
+
+    trashcan_default = "<:trashcan:637136429717389331>"
+    if Client.debug:
+        trashcan_default = "\U0001f5d1\ufe0f"
+    trashcan = environ.get("TRASHCAN_EMOJI", trashcan_default)
```

Comment:
```suggestion
    trashcan = environ.get(
        "TRASHCAN_EMOJI",
        f"\N{WASTEBASKET}" if Client.debug else "<:trashcan:637136429717389331>",
    )
```
Normally an if would probably be nicer, but here ideally we should avoid any extra variables being left over in the namespace so people don't accidentally use them in code or get confused. (alternatively we could prefix the default with a `_` and `del` it afterwards)